aspnetcore/signalr/hubs.md

```diff
@@ -120,6 +120,56 @@ By default, a server hub method name is the name of the .NET method. To change t
 
 :::code language="csharp" source="~/../AspNetCore.Docs.Samples/signalr/hubs/samples/6.x/SignalRHubsSample/Snippets/Hubs/ChatHub.cs" id="snippet_HubMethodName" highlight="1":::
 
+## Inject services into your hub
+
+> [!NOTE]
+> The following feature is only available in .NET 7.0+
+
+Hub constructors can accept services from DI as parameters which can be stored in properties on the class for use in a hub method.
+
+When injecting multiple services for different hub methods or as an alternative way of writing your code, hub methods can also accept services from DI.
+By default hub method parameters will be inspected and resolved from DI if possible.
+
+```csharp
+services.AddSingleton<IDatabaseService, DatabaseServiceImpl>();
+
+// ...
+
+public class ChatHub : Hub
+{
+    public Task SendMessage(string user, string message, IDatabaseService dbService)
+    {
+        var userName = dbService.GetUserName(user);
+        return Clients.All.SendAsync("ReceiveMessage", userName, message);
+    }
+}
+```
+
+If this behavior is undesired it can be disabled with [DisableImplicitFromServicesParameters](xref:signalr/configuration#configure-server-options).
```

Comment:
Prob best to restate the noun over using a pronoun. How about ...

```suggestion
If implicit resolution of parameters from services isn't desired, disable it with [`DisableImplicitFromServicesParameters`](xref:signalr/configuration#configure-server-options).
```

Reply:
I see ... then there's a stray CRLF at the end of Line148.

Reply:
... oh ... Line 640.

Actually, I think without **two spaces** at the end of the line, it will end up rendering as one paragraph anyway ... even if the CRLF stays there. The DocFX trick that we use to merely place a `<br>` is two spaces. Otherwise, I think DocFX is going to make `<p>`s out these two lines.

Reply:
Sounds good. Looks like it's a paragraph on the review site.